src/app/clients/add/databaseFunctions.ts

```diff
@@ -1,11 +1,20 @@
-import supabase, { InsertSchema, Schema } from "@/supabase";
-import { Person } from "@/components/Form/formFunctions";
+import supabase, { InsertSchema, Schema, UpdateSchema } from "@/supabase";
+import { checkboxGroupToArray, Fields, Person } from "@/components/Form/formFunctions";
+import { AppRouterInstance } from "next/dist/shared/lib/app-router-context";
 
 type FamilyDatabaseInsertRecord = InsertSchema["families"];
+type FamilyDatabaseUpdateRecord = UpdateSchema["families"];
+
+export type SubmitFormHelper = (
+    fields: Fields,
+    router: AppRouterInstance,
+    primaryKey?: string
+) => void;
 export type ClientDatabaseInsertRecord = InsertSchema["clients"];
+export type ClientDatabaseUpdateRecord = UpdateSchema["clients"];
```

Comment:
Yee I was referring to the type above - a quick glimpse of `database_types_file.ts` kinda show me they are fairly similar (for client it is identical, for other forms maybe some param become optional in update?) - just thinking is there ways to combine them

Reply:
I think it's better to assume they maynot always be identical in the future